src/inference/hmc.jl

```diff
@@ -470,23 +399,19 @@ function steps!(
     vi,
     samples;
     rng::AbstractRNG=GLOBAL_RNG,
-    verbose::Bool=true,
-    progress::Bool=false
```

Comment:
I believe this is a merge issue, we should not drop progress. Same for all places

Reply:
Not a merge issue, it's a design choice -- see above.